bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/ServiceBean.java

```diff
@@ -17,5 +17,13 @@
     ServiceBean() {
     }
 
+    /**
+     * Service Name
+     */
     public String name;
+
+    /**
+     * Persistence service class
+     */
+    public String classname;
```

Comment:
Yes please - and possibly rename name to id if it does not have too much impact (which I don't think).